ts/src/pro/okx.ts

```diff
@@ -391,22 +437,28 @@ export default class okx extends okxRest {
         const data = this.safeValue (message, 'data', []);
         const marketId = this.safeString (arg, 'instId');
         const market = this.safeMarket (marketId);
-        const symbol = market['id'];
+        const safeMarketId = market['id'];
+        const symbol = market['symbol'];
         const interval = channel.replace ('candle', '');
         // use a reverse lookup in a static map instead
         const timeframe = this.findTimeframe (interval);
         for (let i = 0; i < data.length; i++) {
             const parsed = this.parseOHLCV (data[i], market);
-            this.ohlcvs[symbol] = this.safeValue (this.ohlcvs, symbol, {});
-            let stored = this.safeValue (this.ohlcvs[symbol], timeframe);
+            this.ohlcvs[safeMarketId] = this.safeValue (this.ohlcvs, safeMarketId, {});
```

Comment:
@ttodua why are you indexing by the marketId? we always use symbol as the key

Reply:
it was there, i haven't changed the flow. just renamed the variable which was `symbol` before it, but actually it was market id

Reply:
oh I see, but still I think we should use the symbol